src/include/indexlr-workers.h

```diff
@@ -338,16 +332,22 @@ MinimizeWorker::work()
 
 			auto minimizers = getMinimizers(hashes, w);
 
-			if (stlfr) {
-				std::regex right("^.*#+");
-				std::regex left("/.*");
-
-				read.barcode = std::regex_replace(read.id, right, "");
-				read.barcode = std::regex_replace(read.barcode, left, "");
+			if (read.id.find("BX:Z:") == std::string::npos) {
+				size_t rightPos = read.id.find('#');
```

Comment:
Should `rightPos` be `leftPos`? Better to call it `sharpPos`.

Reply:
Yeah, mixed up left and right.